server/services/helpdesk/ZendeskService.ts

```diff
@@ -44,7 +44,7 @@ Email: ${contactHelpdeskForm.email}
           tags: externalUser ? 'slm_legal_sender' : 'slm_mailroom',
         },
       }
-      const response = (await ZendeskService.restClient().post({
+      const response = (await ZendeskService.restClient().update({
```

Comment:
Woah, hang on, why have these changed to `.update`? This is not an update operation, its a POST for a new entity